hooks/usePoolStats/usePoolStats.ts

```diff
@@ -9,8 +9,8 @@ import {
 } from 'types/generated/graphql/uniswapSubgraph';
 import { useQuery } from 'urql';
 
-const LOADING = 'Loading...';
-const NO_DATA = '---';
+export const POOL_STATS_LOADING = 'Loading...';
```

Comment:
not totally sure about this, but no other way to tell if the returned value is loading or not, and displaying 'Loading...' on the component doesn't look good

Reply:
I think these strings should remain internal, if we need to know whether feeTier is loading we should add a `feeTierLoading` boolean to the return that a consumer can rely on